projects/plugins/jetpack/extensions/blocks/subscriptions/block.json

```diff
@@ -17,14 +17,18 @@
 		"align": [ "wide", "full" ]
 	},
 	"styles": [
+		{
+			"name": "split",
+			"label": "Split",
+			"isDefault": true
+		},
 		{
 			"name": "compact",
 			"label": "Compact"
 		},
 		{
-			"name": "split",
-			"label": "Split",
-			"isDefault": true
+			"name": "button",
```

Comment:
we could call it `button-only`, then we could do `$is_style_button_only = get_attribute( $attributes, 'className' ) === 'is-style-button_only';` 🤓